src/System.Globalization/tests/CompareInfo/CompareInfoTests.Compare.cs

```diff
@@ -320,7 +320,7 @@ public void Compare(CompareInfo compareInfo, string string1, int offset1, int le
         [ActiveIssue(5436, PlatformID.AnyUnix)]
```

Comment:
5436 is closed... and about TcpClient...

I think the comment was right, and this is wrong, 5463 is "Investigate failing System.Globalization.Tests on Unix"